test/TestLogger.UnitTests/TestDoubles/JsonTestResultSerializer.cs

```diff
@@ -86,8 +88,6 @@ private Test CreateTest(TestResultInfo result)
         {
             return new ()
             {
-                FullyQualifiedName = result.TestCase.FullyQualifiedName,
-                DisplayName = result.TestCase.DisplayName,
```

Comment:
On a second thought, I think we should keep this `result.TestCaseDisplayName`. It will help us easily observe the diffs between method/display names for each of the adapters the snapshots. What do you think?

Reply:
Thats fine. Added internals visible to, and put this back